### PR TITLE
Add ticket info

### PR DESCRIPTION
**Pull request by patricksanders**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -95,15 +99,26 @@ googleAnalytics = ""
...
+
+## Ticket Information
+
+We're doing tickets a little differently this year. As always, BSides Atlanta 2019 is a free event that is open to all. This year we've added a Personal Sponsor option. This sponsorship helps pay for putting the conference on and gets you early access to a ticket.
+
+### Ticket Schedule
+
+- Monday, February 18, Noon EST - Personal Sponsor tickets available
+- Friday, March 1, Noon EST - Wave 1 of GA tickets available
+- Monday, April 1, Noon EST - Wave 2 of GA tickets available
```
Comment (brimstone) [inline config.toml:123]: April Fools! I like it on a different day than the first wave.

Comment (patricksanders) [reply config.toml:123]: Me too! Hopefully it'll give more people a chance to get the tickets.

File: config.toml [hunk 2]
```diff
@@ -112,6 +112,10 @@ Each BSides is a **community-driven** framework for building events for and by i
 
 We're doing tickets a little differently this year. As always, BSides Atlanta 2019 is a free event that is open to all. This year we've added a Personal Sponsor option. This sponsorship helps pay for putting the conference on and gets you early access to a ticket.
 
+We also decided to release free GA tickets in waves based on feedback from all of you. Check the schedule below and get your clicking fingers ready!
+
+[Click here](https://ti.to/bsidesatl/bsidesatl2019) to get your tickets.
```
Comment (brimstone) [inline config.toml:117]: I like the more specific calls to action like "Get your [tickets] now" instead of "Click here" I'm find either way though.

**Review by brimstone — APPROVED** [review]
LGTM

File: config.toml (hunk 2 above)
Comment (patricksanders) [reply config.toml:117]: I'm gonna leave it like this for now because links aren't obvious with our style, and I don't know how to CSS very well

**Pull request merged**